lib/core/mount.js

```diff
@@ -168,9 +169,13 @@ function mount(mountPath, parentApp, events) {
 
 		// Initialize the session store
 		try {
-			
-			var SessionStore = require(sessionStore)(express);
-			
+      if(sessionStore === 'connect-redis') {
+				expressSession = require('express-session');
```

Comment:
There aren't really any defaults you'd wanna use. I guess we could put localhost and the port that redis uses by default (which I don't remember) in there. But in production you're going to have to set all of these with environment variables, regardless, so setting defaults would only affect developers on their local machine. And, that seems like the worst thing to do.